rpa-builder/6.6/modules/ROOT/pages/toolbox-mulesoft-idp-submit-document-to-mulesoft-idp.adoc

```diff
@@ -0,0 +1,60 @@
+= Submit Document to MuleSoft IDP 
```

Comment:
This heading implies the topic is a task, but it seems to be more reference. Any way to make that more clear?